### DISCUSSION TITLE
Strange alert Measurement for 'resolve-async:/package.json:/' was requested, but never was started

### DISCUSSION THREAD

**Discussion by ofergoli** [codesandbox/codesandbox-client#5905] [Q&A]
![image](https://user-images.githubusercontent.com/8341557/122097720-aa803d00-ce18-11eb-9007-d10437fe5f67.png)
Does someone know what is that alert?

**Comment by ofergoli** [comment]
It's on embed sandbox

**Comment by joakimc** [comment]
I'm also seeing this. Did you find out what's causing it?

**Reply by ofergoli** [reply]
Yes, remove the @babel/core dependency from devDependencies in package.json. (:
